editor/import/scene_importer_mesh.cpp

```diff
@@ -261,6 +266,103 @@ void EditorSceneImporterMesh::clear() {
 	mesh.unref();
 }
 
+void EditorSceneImporterMesh::create_shadow_mesh() {
+	if (shadow_mesh.is_valid()) {
+		shadow_mesh.unref();
+	}
+
+	shadow_mesh.instance();
```

Comment:
I think `.instance()` should be called after the blendshape/skeleton checks. Otherwise, in line 254 of this same file, we check for `shadow_mesh.is_valid()` which will be true, but it will contain an empty mesh.

Not sure of the implications of this deeper in the rendering code, but I feel like things can break.

Reply:
ah good point, changing it